test/aries-js-worker/test/didexchange/didexchange_e2e.js

```diff
@@ -82,11 +82,15 @@ export const didExchangeClient = class {
     }
 
     static async addRouter(mode, agent) {
-        let resp = await agent.mediator.getConnections()
-        if (resp.connections){
-            for (let i = 0; i < resp.connections.length; i++) {
-                await agent.mediator.unregister({"connectionID": resp.connections[i]})
+        try {
+            let resp = await agent.mediator.getConnections()
+            if (resp.connections){
+                for (let i = 0; i < resp.connections.length; i++) {
+                    await agent.mediator.unregister({"connectionID": resp.connections[i]})
+                }
             }
+        }catch (e){
+            console.log(e)
```

Comment:
maybe mention this is a warning, not a breaking error..

Reply:
added condition to handle an error properly

Reply:
nitpick: you can still log the `not found error` as a warning